pyecoregen/cli.py

```diff
@@ -53,10 +54,19 @@ def configure_logging(parsed_args):
     )
 
 
+def select_uri_implementation(ecore_model_path):
+    """Select the right URI implementation regarding the Ecore model path schema."""
+    url_pattern = re.compile('^http(s)?://.*')
```

Comment:
Just a minor: I suggest to make this a module level constant ``URL_PATTERN``, so the compilation will happen only once during import.

Feel free to merge yourself, since I am offline for the rest of the day.

Thanks for leeting me know about the change,
Mike